### PR TITLE
Feature/update l 10

### PR DESCRIPTION
**Pull request by rastik1584**

### CODE REVIEW

File: tests/TestCase.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ protected function tearDown(): void
     /**
      * @param \Illuminate\Foundation\Application $app
      */
-    protected function getEnvironmentSetUp($app)
+    protected function getEnvironmentSetUp($app): void
```
Comment (timoransky) [inline tests/TestCase.php:38]: why there is no type for parameter?

File: composer.json
Comment (timoransky) [inline composer.json:1]: This will remove support for Laravel 9 which I don't think is ideal, since I'm pretty sure we can keep compatibility with both Laravel 9 and 10, just like we do with Craftable PRO

**Review by timoransky — COMMENTED** [review]
I like the usage of the native imports, but in that case, we can remove the old definitions. Also, let's discuss the composer.json because I don't like removing support for Laravel 9... I don't think there is a need to be this radical.

File: tests/TestCase.php (hunk 1 above)
Comment (rastik1584) [reply tests/TestCase.php:38]: Must be compatible with Orchestra\Testbench\TestCase getEnviromentSetUp and getPackageProviders

**Comment by KirePetrov** [comment]
@rastik1584 @timoransky
Hi guys, is this in the plan to merge it in near future?
This one package stops me to upgrade to Laravel 10.x almost a year.
Thanks.

File: composer.json [hunk 2]
```diff
@@ -16,16 +16,16 @@
...
     ],
     "require": {
-        "php": "^8.0.2",
-        "illuminate/support": "^9.0",
-        "illuminate/http": "^9.0",
-        "monolog/monolog": "^1.12|^2.0",
+        "php": "^8.1.2",
+        "illuminate/support": "^9.21|^10.0",
+        "illuminate/http": "^9.21|^10.0",
+        "monolog/monolog": "^3.3",
```
Comment (timoransky) [inline composer.json]: @rastik1584 is this backwards compatible as well? Have you tried?

File: composer.json [hunk 3]
```diff
@@ -16,16 +16,16 @@
         }
     ],
     "require": {
-        "php": "^8.0.2",
-        "illuminate/support": "^9.0",
-        "illuminate/http": "^9.0",
-        "monolog/monolog": "^1.12|^2.0",
+        "php": "^8.1.2",
```
Comment (timoransky) [inline composer.json]: Why did you change the PHP as well? I have a feeling this will break Laravel 9 compatibility.

File: composer.json [hunk 4]
```diff
@@ -16,16 +16,16 @@
...
+        "illuminate/support": "^9.21|^10.0",
+        "illuminate/http": "^9.21|^10.0",
```
Comment (timoransky) [inline composer.json]: There should not be a minor version declared. what if someone has a Laravel 9 project with version lower than 9.21? I know, it's kinda edge case, but you get the point. This would not work for him

**Review by timoransky — CHANGES_REQUESTED** [review]

**Review by timoransky — APPROVED** [review]
